packages/stable/src/api/documents/types.gen.ts

```diff
@@ -272,7 +302,7 @@ export interface DocumentGeoJsonGeometry {
      * Coordinates of the geometry.
      * @example [10.74609,59.91273]
      */
-    coordinates?: any[];
```

Comment:
Looks like this is a breaking change. but then again, you should never be able to put anything but numbers in there I believe. Perhaps a patch is valid for this case.

Reply:
This is infact a breaking change according to the tests.
I think we could just revert this back for now and avoid any breaking changes in this PR